glossary.md

```diff
@@ -1,16 +1,80 @@
 You may also need to check https://chromium.googlesource.com/chromiumos/docs/+/master/glossary.md.
 
-* LGTM: "Looks good to me", commonly used to approve a code review.
-* PTAL: Please take a look.
-* RSLGTM: "Rubber-stamp looks good to me". The reviewer approving without doing
+* ABI: [Application Binary Interface](https://en.wikipedia.org/wiki/Application_binary_interface)
+* ASAP: As Soon As Possible.
+* ASYNC: Asynchronous
+* BE: Big Endian - Byte order style in a multibyte data.
+  (Opposite of LE, Little Endian)
+* CI: Continuous Integration.
+* CITGM: Canary In The Gold Mine - a smoke test unit in the repo that
+  contain popular npm modules.
+* CJS: Common JS.
+* CLDR: Common Locale Data Repository.
+* CLI: [Command Line Interface](https://en.wikipedia.org/wiki/Command-line_interface)
+* CMD: Command.
+* CVE: Common Vulnerebilities and Exposures - A database that
+  maintains reported security exposures.
+* ECMA: European Computer Manufacturers Association - A body that
+  governs JS spec among other things.
+* ENV: Environment - General term for execution environment.
+* EOF: [End Of File](https://en.wikipedia.org/wiki/End-of-file)
+* EOL: End Of Line (when used within a program), End of Life
+  (when used within project documents).
+* ESM: ECMA Script Module.
+* ETW: Event Tracing for Windows.
+* FD: File Descriptor.
+* FFDC: First Failure Data Capture - Common terms for logs, traces
+  and dumps that are produced by default on program error.
+* FIPS:  Federal Information Processing Standards.
+* FS: File System.
+* ICU: International Components for Unicode.
+* IPC: Inter Process Communication.
+* JIT: Just In Time - General term for dynamic compiler in
+  managed runtimes.
+* JS/C++ boundary: A layer that bridges between JS APIs and the C++
+  helpers that implements / abstracts platform capabilities.
+* LGTM: Looks good to me - commonly used to approve a code review.
+* LTS: Long Term Support.
+* MDN: Mozila Development Network - A vast and authentic
+  documentation on JavaScript.
+* OOB: Out Of Bounds - Used in the context of TCP data transport.
+* OOM: Out Of Memory.
+* OSX: Mac OS.
+* PPC: Power PC.
+* PTAL: Please Take A Look.
+* RAII: [Resource Aquisition Is Intialization](https://en.wikipedia.org/wiki/Resource_acquisition_is_initialization)
+* REPL: Read Evaluate Print Loop.
```

Comment:
This does not really explain what it is in the context of Node.js - I think people who don't know what REPL might still be puzzled after reading this.

Reply:
Explained it. PTAL.